src/elements/content-sidebar/BoxAISidebar.tsx

```diff
@@ -18,32 +20,66 @@ import { mark } from '../../utils/performance';
 
 import messages from '../common/messages';
 import sidebarMessages from './messages';
+import './BoxAISidebar.scss';
 
 const MARK_NAME_JS_READY: string = `${ORIGIN_BOXAI_SIDEBAR}_${EVENT_JS_READY}`;
 
 mark(MARK_NAME_JS_READY);
 
 export interface BoxAISidebarProps {
+    onClearClick: () => void;
     onExpandClick: () => void;
 }
 
-function BoxAISidebar({ onExpandClick }: BoxAISidebarProps) {
+function BoxAISidebar({ onClearClick, onExpandClick }: BoxAISidebarProps) {
     const { formatMessage } = useIntl();
 
-    return (
-        <SidebarContent
-            actions={
+    const renderBoxAISidebarTitle = () => {
+        return (
+            <div className="bcs-BoxAISidebar-title-part">
+                <Text as="h3" className="bcs-title">
+                    {formatMessage(messages.sidebarBoxAITitle)}
+                </Text>
+                <BoxAiAgentSelector
+                    agents={[]}
+                    onErrorAction={() => null}
+                    requestState={REQUEST_STATE.SUCCESS}
+                    selectedAgent={null}
+                    triggerChipClassName="sidebar-chip"
+                />
```

Comment:
can we set `shouldHideAgentSelectorOnLoad` here for consistency with modal?

In general we should try to keep the props passed here consistent with the usage in BoxAiContentAnswers as much as possible: https://git.dev.box.net/Box/frontend-mono/blob/aca02607f44492822a372efe33a60a97ace03822/packages/group-shared-features/box-ai-content-answers/src/lib/components/modal/modal.tsx#L165

Reply:
@DanilaRubleuski yes should switch to using `BoxAiAgentSelectorWithApi` once add the context provider. You wrap it in the `AgentsContext` provider and that will add automatic state handling for the selector. Fine the leave the props as is for now then and address in follow up PR